lib/hubstats/github_api.rb

```diff
@@ -98,7 +98,8 @@ def self.update_teams
       grab_size = 250
       begin
         client = Hubstats::GithubAPI.client
```

Comment:
I think its because calling `Hubstats::GithubAPI.client` initializes a new client and `Octokit.auto_paginate` has to be set before initialization.  Try moving it first and reusing the `client` variable as before.